src/main/webapp/resources/js/apis/pipelines/pipelines.js

```diff
@@ -62,3 +62,9 @@ export const launchPipeline = (id, parameters) =>
     .catch((error) => {
       throw new Error(error.response.data);
     });
+
+export function saveNewPipelineParameters({ label, parameters, id }) {
```

Comment:
This should probably catch and rethrow the error if there was an error saving the params

Reply:
What are you talking about, no errors ever happen.  Updated in 9230a9b964afe32d63260345073a932dba82c78f